coral/README.md

```diff
@@ -24,10 +24,22 @@
 
 ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) it.
 
+### Usage: How to run Coral inside the Klaw application
+
+1. navigate to this directory
+2. run `pnpm install`
+3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also add the current Coral build files in the right place.
+4. go to the root directory and follow the [instructions on how to run Klaw](../README.md#Install)
+
+➡️ Klaw will run in `http://localhost:9097` (note: it's `http` instead of `https`!)
```

Comment:
Can you tell me where they can see this? (`applications.properties` ?). As a no-Java-experience user :D I didn't know that. 
So it may be helpful to add that in case that someone wants to contribute on the FE and run Klaw, but is also missing that background.

Reply:
If they enable these properties, server would run on https